apps/ledger-live-mobile/src/components/Graph/Placeholder.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Flex, Text } from "@ledgerhq/native-ui";
+import { Image } from "react-native";
+import { useTranslation } from "react-i18next";
+
+const graphPlaceholderSource = require("../../../assets/images/graph/graphPlaceholder.png"); // eslint-disable-line @typescript-eslint/no-var-requires
```

Comment:
@KVNLS Can you please advise :pray: is this the best way to import and use a static asset? Thanks

Reply:
Changing it to `import` statement, but no paths aliases are provided in tsconfig for ledger-live-mobile

Reply:
done, thanks